presidio-image-redactor/tests/integration/test_dicom_image_pii_verify_engine_integration.py

```diff
@@ -49,7 +34,7 @@ def test_verify_correctly(
         test_image,
         test_ocr_results,
         test_analyzer_results,
-    ) = mock_engine.verify_dicom_instance(get_mock_dicom_instance, PADDING_WIDTH)
+    ) = DicomImagePiiVerifyEngine().verify_dicom_instance(get_mock_dicom_instance, PADDING_WIDTH)
```

Comment:
Why using a fixture isn't the right approach in this case?

Reply:
Doesn't really matter, IMO it's simpler that way since it reduces method signatures. 
Initially removed it since I thought that the bug is there

Reply:
If this is used more than once, then reloading it would likely make tests slower (image analyzer loads text analyzer which loads spacy)

Reply:
We instantiate each time in the fixture case, anyway we want to start fresh in each scenario

Reply:
We instantiate the test, but not the fixture (which is only loaded once). Or am I missing something?

Reply:
The fixture is loaded twice, thus the class is instantiated twice. 
The scope of the fixture is 'function' so it is called with every test function (try to run the file and set a BP inside the fixture).
This is very much intentional since its a good practice to reset all your mocks and stubs between tests (so that one won't affect the other). Otherwise, it could have been a global / module level instantiation that is used directly in the tests

Reply:
ok, thanks for clarifying. As those objects are stateless, there shouldn't be a dependency between tests but as long as it's not a performance overhead we can keep it as is.